nilearn/connectome/group_sparse_cov.py

```diff
@@ -205,9 +203,7 @@ def group_sparse_covariance(subjects, alpha, max_iter=50, tol=1e-3, verbose=0,
 
     References
     ----------
-    .. [1] Jean Honorio and Dimitris Samaras, "Simultaneous and Group-Sparse
-       Multi-Task Learning of Gaussian Graphical Models". arXiv:1207.4255
-       (17 July 2012). http://arxiv.org/abs/1207.4255.
+    .. footbibliography::
```

Comment:
I thought this was a mistake since the ArXiV title doesn't match the bibtex title ("Simultaneous and Group-Sparse Multi-Task Learning of Gaussian Graphical Models" vs "On the Statistical Efficiency of l1,p Multi-Task Learning of Gaussian Graphical Models") but this is indeed the bibtex ref provided on arXiV...

Reply:
Oh yes, I should have commented on this. The title has changed from the first (2012) to the second (2015) version of the preprint. I thought it would make sense to cite the latest version, especially since it had already been referenced elsewhere. Thanks!

Reply:
>I thought it would make sense to cite the latest version

Absolutely! I think we can merge this, thanks!